cluster-provision/images/vm-image-builder/example/image-url-s390x

```diff
@@ -0,0 +1 @@
+https://archives.fedoraproject.org/pub/fedora-secondary/releases/39/Cloud/s390x/images/Fedora-Cloud-Base-39-1.5.s390x.qcow2
```

Comment:
I think the x86 and s390x images should align on the fedora versions as these images are used for testing - we should try to reduce the differences wherever we can.

Reply:
Hi @brianmcarey , actually when Iam going through the fedora 35 I see it is EOL since 2022-12-13. So, I am thinking it is better to use fedora39 or fedora 40. So, in order to align on the x86 and s390x fedora versions, Can I update the x86 also to fedora 39? will that be fine? 
Another thing is Iam not able to find fedora 35 for s390x in the fedora's official public archives. 
I am only able to find the fedora for s390x from fedora 39 onwards.
See here:
https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/35/Cloud/
https://archives.fedoraproject.org/pub/fedora-secondary/releases/35/
I am able to find the fedora35 in some insecure URL i.e.
https://download4.fedora.redhat.com/pub/archive/fedora-secondary/releases/35/Cloud/s390x/images/
I am not sure whether to use it or not?

Reply:
Lets go with fedora39 for now. The x86 images will have to be updated in order to align. I will try to build you a test image.

Reply:
sure thanks

Reply:
Heres a test image: 

`quay.io/bcarey/fedora-with-test-tooling:v20240709-1ce2923`

Can you try it out before we merge this?

Reply:
sure @brianmcarey